core/admin/admin.php

```diff
@@ -731,6 +737,14 @@ public function admin_action_new_post() {
 		die;
 	}
 
+	private function sanitize_post_data( $post_data ) {
+		$post_data = [
+			'post_title' => isset( $post_data['post_title'] ) ? sanitize_text_field( $post_data['post_title'] ) : '',
+			'post_content' => isset( $post_data['post_content'] ) ? sanitize_textarea_field( $post_data['post_content'] ) : '',
+			'post_status' => isset( $post_data['post_status'] ) ? $post_data['post_status'] : 'draft',
+		];
```

Comment:
So can we only save these 3 keys? What about post_type, post_excerpt etc?

Reply:
Post type is added later in the main method, i am not sure about post_excerpt exactly

Reply:
Let's get an allowed list of fields per role, and just remove any fields that are not in the whitelist.